pages/Contact.tsx

```diff
@@ -0,0 +1,111 @@
+
+export default function Home() {
+  const [dropDownValue, setDropDownValue] = useState<
+    ContactSubject | undefined
+  >(undefined);
+
+  const handleSubmit = async (
+    formData: Record<string, string>
+  ): Promise<SubmissionResponse> => {
+    const validFormData = processFormData(formData);
+    if (validFormData == undefined) {
+      // Invalid Form Submission
+      return {
+        success: false,
+        status: 500, // Unknown Response
+        message: `Impossible Invalid Payload Provided, Please Contact ${website_config.email}.`,
+      };
+    } else {
+      // Submit Our Form
+      try {
+        const response = await fetch('/api/submitForm', {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(validFormData),
+        });
+        // Build A Blanket Response
+        const submissionResponse: SubmissionResponse = {
+          success: false,
+          status: response.status,
+          message: 'An unknown error occurred. Please try again.',
+        };
+        // Process the response
+        if (response.status == 200 && response.ok) {
+          const data = await response.json();
+          if ('message' in data && typeof data.message === 'string') {
+            submissionResponse.success = true;
+            submissionResponse.message = data.message;
+          }
+        }
+        return submissionResponse;
+      } catch (e) {
+        console.error('Failed to submit form, unhandled error', e);
+        return {
+          success: false,
+          status: 500, // Unknown Response
+          message: 'An unknown error occurred. Please try again.',
+        };
+      }
+    }
+  };
+
+  return (
+    <>
+      <EventBanner />
+      <NavBar currentPage='Contact Us' />
+      <section className={styles.container}>
+        <main className={styles.MainArea}>
+          <BlurBanner
+            imagePath='/FaryonBridge.jpg'
+            altText='About Us Image'
+            title={website_config.title}
+            sectionHeader='Positions'
+          />
+          <ListingsSection
+            positions={contact_page.listings}
+            setDropDownValue={setDropDownValue}
+          />
+          <SectionHeader
+            title='Become a Sponsor'
+            style={SectionHeaderStyle.primary}
+            location={SectionLocation.RIGHT_ALIGNED}
+          />
+          <div className={styles.sponsorButton}>
+            <TextSection
+              section={contact_page.sponsor_section}
+              style={TextSectionStyle.primary}
+            />
+          </div>
+          <SectionHeader
+            title='Contact Us'
+            style={SectionHeaderStyle.secondary}
+            location={SectionLocation.LEFT_ALIGNED}
+          />
+          <ContactSection
+            dropDownValue={dropDownValue}
+            onSubmit={handleSubmit}
+          />
```

Comment:
Figure i should ask even though i really dont wanna but should we take this from config

Reply:
I dont think we need to although, we should probably be using `Section` at least over just building the sections manually.